docs/settings.md

```diff
@@ -1,16 +1,54 @@
 ## Pageserver
 
-### listen_pg_addr
+Pageserver is mainly configured via a `pageserver.toml` config file.
+If there's no such file durinig `init` phase of the server, it crates the file itself, otherwise it does not alter and parses it.
 
-Network interface and port number to listen at for connections from
-the compute nodes and safekeepers. The default is `127.0.0.1:64000`.
+There's a possibility to pass an arbitrary config value to the pageserver binary as an argument: such values override
+the values in the config file, if any are specified for the same key and get into the final config during init phase.
 
-### listen_http_addr
 
-Network interface and port number to listen at for admin connections.
-The default is `127.0.0.1:9898`.
+### Config example
 
-### checkpoint_distance
+```toml
+# Initial configuration file created by 'pageserver --init'
+
+listen_pg_addr = '127.0.0.1:64000'
+listen_http_addr = '127.0.0.1:9898'
+
+checkpoint_distance = '268435456' # in bytes
+checkpoint_period = '1 s'
+
+gc_period = '100 s'
+gc_horizon = '67108864'
+
+open_mem_limit = '134217728' # in bytes
```

Comment:
This parameter is not used anywhere, should we remote it?

Reply:
yeah